### DISCUSSION TITLE
NameError: name 'show_subject' is not defined

### DISCUSSION THREAD

**Discussion by ruby0101** [TorchIO-project/torchio#426] [Q&A] [answered]
Hi, I have just followed up a tutorial (https://colab.research.google.com/github/fepegar/torchio-notebooks/blob/main/notebooks/TorchIO_tutorial.ipynb#scrollTo=wqX8VOOYpQDb). Nothing is different. But it gives me 'show_subject' is not defined error. Could you check it please?

**Comment by fepegar** [comment, marked as answer]
That is defined in a hidden cell, you need to execute it. Right under "**Define some visualization functions:**".

But I'm actually modifying that tutorial at the moment and that cell will not be needed anymore, so you might want to check again in a couple of hours.

**Reply by ruby0101** [reply]
Thank you so much.
